docroot/modules/humsci/hs_theme_helper/hs_theme_helper.module

```diff
@@ -0,0 +1,77 @@
+<?php
```

Comment:
This second line functionality is something we have seen in past themes. Wouldn't it make sense for this to be in stanford_basic?

Reply:
I thought about that but you can't alter the form from the theme.

Reply:
Instead of altering the form, what about creating your own settings much like how OFW did it. 

You override the title and add a second line if available from the theme settings.

Reply:
Then would we have to move all of the basic site settings into the theme form? such as the home page, 404, 403 and slogan?

Reply:
No, just the titles. You have access to the templates and can apply the correct logic. 

You can swap out `{{ site_name }}` for the theme setting value and add `{{ site_name_line_2 }}` in the twig template.

Reply:
But then they would have 2 forms to go to. basic site settings for 404, home etc and theme settings for site title.

Reply:
hmm, i need to think about this. it seems like it should go with the theme but i agree, two separate forms doesn't make sense.

Reply:
Moved possible work to https://github.com/SU-SWS/stanford_basic/pull/74